knot.x/knotx/conf/application.conf

```diff
@@ -0,0 +1,12 @@
+########### Modules to start ###########
+# Modules map specify a list of verticles to be started by Knot.x.
+# Each line should have a form of <alias>=<verticle-class-name>
+# where alias is just a name that you can use later in order to define configuration for the module
+# verticle-class-name is a fully qualified class name of the verticle.
+#
+# This JSON object is filled in included files.
+modules {}
+
+include required(classpath("profiles/profile-active.conf"))
```

Comment:
I am wondering what more we can do with `active` profile during performance tests. i assume that we have many scenarios and we want to store all of the in one Knot.x Docker image. Then we can build image with symbolic link`profile-active.conf` that points to real profile like `profile-scenario0.conf`. I am not sure if we can do it dynamically - without rebuilding the image between scenarios, see also https://stackoverflow.com/questions/30494050/how-do-i-pass-environment-variables-to-docker-containers.

Reply:
Would env variables work here?
Maybe we could pass the active profile conf file name with `environment` via docker-compose config?

Reply:
It is kind of misunderstanding. I used wrong name here. In fact the only thing I need is to provide somehow the mocks domain. It must be provided from external file as the property name is fixed: *config.server.options.config.mocks.domain* 

So I moved the file to routes directory with name _mocks.conf_